### DISCUSSION TITLE
License question

### DISCUSSION THREAD

**Discussion by dbelyaev** [orhun/git-cliff-action#31] [General]
Is there any specific reason that git-cliff-action is issued under [GPL v3](https://github.com/orhun/git-cliff-action?tab=GPL-3.0-1-ov-file#readme) license, while git-cliff cli tool is under [Apache-2.0 and MIT](https://github.com/orhun/git-cliff/blob/main/LICENSE-APACHE)?

**Comment by orhun** [comment]
Ah yeah, good point. I changed `git-cliff`'s license in https://github.com/orhun/git-cliff/issues/165 but forgot to make the change here 💀 

I should probably make the change here too (by pinging to contributors etc.) - not sure if it is a big deal though.

Do you have a specific use case that is blocked by GPLv3?

**Reply by dbelyaev** [reply]
No, it is not blocking us from use.

I was investigating licenses for Git-cliff and Git-cliff-actions to see if we can use them in our team. 
While doing so, I came across a license type mismatch between the two related projects and was curious to ask a question. :)

**Reply by orhun** [reply]
I see :)

Created https://github.com/orhun/git-cliff-action/issues/32

**Comment by orhun** [comment]
Re-licensed now!
